src/doc/reference/src/visibility-and-privacy.md

```diff
@@ -40,7 +40,10 @@ With the notion of an item being either public or private, Rust allows item
 accesses in two cases:
 
 1. If an item is public, then it can be used externally through any of its
-   public ancestors.
+   accessible ancestors (either by anyone that has an access to an ancestor
+   in the case of a chain of public ancestors,
+   or by the siblings of a private ancestor, in the case of a chain of public
+   ancestors terminating into a private ancestor).
```

Comment:
`fn hello()` is a public item. It is contained in module `inner` which is private. The module `inner` is contained within module `module`. `hello` is usable from `module`, but this is not directly evident from the specified rules, because they mention only "public ancestors", but here the ancestor is private. I know the intent of the rules, but they could specify it better.